manual/src/refman/expr.etex

```diff
@@ -668,6 +938,16 @@ the array denoted by @expr_1@, replacing element number @expr_2@ by
 the value of @expr_3@. The exception "Invalid_argument" is raised if
 the access is out of bounds. The value of the whole expression is @'()'@.
 
+\begin{caml_example}{toplevel}
+let mul arr n =
+  for x = 0 to Array.length arr - 1 do
+    arr.(x) <- arr.(x) * n
+  done
+
+let x = [|1; 10; 100|]
+let _ = mul x 2;;
```

Comment:
Ah, it does. Because by the time the output is printed, the mutation has already happened:
```
# let scale arr n =
    for x = 0 to Array.length arr - 1 do
      arr.(x) <- arr.(x) * n
    done

  let x = [|1; 10; 100|]
  let _ = scale x 2;;
val scale : int array -> int -> unit = <fun>
val x : int array = [|2; 20; 200|]
```

Reply:
Note that you can add `;;` after the definition of `x`  and then show the value of `x` after calling `scale`.